CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing guidelines
+
+A single gazetteer is a collection of places that are thematically related or relevant to a particular map styling task. Keeping the content of a gazetteer focused and clearly defined will help map designers pick suitable gazetteers for reviewing their map styling projects.
+
+### Choosing places to add to a gazetteer
+
+Typically, you’ll want to pick places that clearly display either:
+
+- A single map feature (e.g. a tunnel)
+- A cluster or group of map features (e.g. several tunnels, surface roads, and bridges)
+- Map features which interact with each other (e.g. a tunnel under a park)
+
+### Considerations for choosing places
+  
+- [ ] Is it **famous or notable**?
+- [ ] Is it relatively **permanent and stable**?
+- [ ] Is it one of the following?
+    1. An **archetypal** example of a map feature or map feature interaction?
+    2. An **extreme** example of a map feature or map feature interaction (in size, density, etc)?
+    3. An **unusual** example of a map feature or map feature interaction (in shape, location relative to other features, etc)?
+- [ ] Do you have reliable [**ground truth**](https://en.wikipedia.org/wiki/Ground_truth) information?
+
+## Mapbox Streets gazetteers
+
+Gazetteers designed for reviewing styles using Mapbox Streets data can go in the `/mapbox-streets` directory.
+
+### Suggestions for places’ properties values
+Refer to the gazetteer spec for properties available to each place entry.
+
+#### place_name
+Choose a name that is as specific as possible to the map feature(s) displayed. For context, it is often helpful to also include the geographic entity one or so level above the specific map feature, e.g. “Jane Byrne Interchange, Chicago", “Central Park, New York City”, or "New York City, United States".
+
+#### zoom
+Use integer zoom levels.
+
+#### description
+Explain why this particular place is significant in the context of this gazetteer.
+
+#### tags
+Optionally include arbitrary key/value tags to capture useful distinctions between places that are not already represented in the data-specific information (aka "highlights" – see below).
+
+#### highlights
+Mapbox Streets gazetteers should as much as possible include data-specific information ("highlights") in each entry.
```

Comment:
I don't understand, intuitively, why these data tags are called, 'highlights.' If this descriptor isn't a commonly used word in this context, I would maybe: 
- start with defining what a 'highlight' is
- how highlights should be used
As the writing is, you start with how 'highlights' should be used and embed the definition within the intended use statement, which is a bit confusing.